src/persist-client/src/internal/state.proto

```diff
@@ -60,9 +58,40 @@ message ProtoStateRollup {
     string diff_codec = 5;
     uint64 seqno = 6;
     uint64 last_gc_req = 10;
+    map<uint64, string> rollups = 12;
```

Comment:
Funny that `ProtoStateRollup` now has a `rollups` field. Maybe the name is not accurate anymore. 😅